lib/modules/accountSwitcher.js

```diff
@@ -28,12 +23,12 @@ module.options = {
 	},
 	accounts: {
 		type: 'table',
-		addRowText: '+add account',
+		addRowText: i18n('accountSwitcherAddAccount'),
```

Comment:
In that case, this PR doesn't have much sense at the moment. Please let me know when you're gonna be ready with these changes in module builder and I'd be happy to make changes to my PR.

(or maybe you're okay with this for the time being and you'd like it to be merged? Up to you of course :) )